config/initializers/trusted_proxies.rb

```diff
@@ -0,0 +1,11 @@
+# Extend default list of trusted proxies with generic private and cloudflare proxy list
+
+# Cloudflare proxies list
+# config/cloudflare_ips.yml fetches every time you build an image. Check Dockerfile l54, l55
+cloudflare_ips = File.read('config/cloudflare_ips.yml').split(/\R+/)
+
+# reserved for private networks IPs according to https://whatismyipaddress.com/private-ip
+private_ips = ["10.0.0.0/8","172.16.0.0/12","192.168.0.0/16"]
```

Comment:
Those ranges are already present in ActionDispatch::RemoteIp::TRUSTED_PROXIES

Reply:
@calj updated